src/zeroband/train.py

```diff
@@ -192,7 +192,7 @@ def train(config: TrainingConfig):
         batch_size=config.optim.batch_size * config.optim.step_per_rollout,
         data_config=config.data,
         step_count_init=step_count_init,
-        use_vllm_logprobs=config.recompute_logprobs,
+        use_vllm_logprobs=True,
```

Comment:
eh so always use vllm logprobs now?

Reply:
this is just for dataset, so we always load the logprob for logprob, but we then later decide if we override them or not